src/_pytest/mark/expression.py

```diff
@@ -198,13 +199,13 @@ class Expression:
     def __init__(self, code: types.CodeType) -> None:
         self.code = code
 
-    @classmethod
-    def compile(self, input: str) -> "Expression":
+    @classmethod  # noqa: A003
```

Comment:
Same comment here regarding `property`.

Reply:
resolving as per my comment below your review. Please reopen if you disagree